lms/djangoapps/grades/models.py

```diff
@@ -380,6 +381,7 @@ class Meta(object):
         # (user_id) for course dashboard; explicitly declared as an index below
         unique_together = [
             ('course_id', 'user_id'),
+            ('date_passed', 'course_id'),
```

Comment:
I don't think so.  We already have course_id/user_id indexed together, so if you try to get the passed_timestamp for a particular user, it will use that index to find it instead.  Each user will only have a single course grade per course, so the passed_timestamp is irrelevant. 

Note the related comments, that this was supposed to be `indexed_together`, not `unique_together`.